info.limpet.rcp/src/info/limpet/rcp/data_provider/data/CollectionWrapper.java

```diff
@@ -18,42 +18,6 @@ public CollectionWrapper(final LimpetWrapper parent,
 		_collection = collection;
 	}
 
-	
```

Comment:
closing this discussion. 

improved equals & hashcode have been moved to a separate issue:  #180